### PR TITLE
crawl deeper

### PR DESCRIPTION
**Pull request by caydijay**
Testing on facebook/react.
Will only merge after verifying the impact these changes made.
Will post before / after stats here.

### CODE REVIEW

File: statstool [hunk 1]
```diff
@@ -35,18 +35,16 @@ set -eu -o pipefail
     }
 }
 
-PR_SAMPLESIZE=250
-COMMIT_PR_SAMPLESIZE=200
+PR_SAMPLESIZE=2500
```
Comment (gerner) [inline statstool:38]: I think 250/200 is fine for these samples. The problem with increasing this is that each request is one API request done serially. if you increase this 10-fold I bet you will see the crawler running _much_ slower. And this sample is really only there to estimate if our heuristic link between commits and PRs (merge sha in PR + autolink patterns in commit messages) is good. Having 250/200 is sufficient for that.

**Review by gerner — CHANGES_REQUESTED** [review]
take a look at those sample sizes and my argument to keep them small.

File: statstool (hunk 1 above)
Comment (caydijay) [reply statstool:38]: I will revert to 250/200.

**Comment by caydijay** [comment]
* 60 out of 111 pages pulled:
  * isolated: 3:15
  * total: 4635s
    * done preparing code history in 56s.
    * done fetching pull request and issue data in 4292s.
    * done pulling commit pull request info in 74s.
    * done collecting sample PR data in 213s.
    * exceeded max retry count 8 on https://api.github.com/repositories/10270250/issues/comments?since=2017-04-19T16%3A42%3A56Z&sort=created&order=desc&per_page=100&page=2

* 111 out of 111 pages pulled:
  * isolated: 05:53
  * total: 8675s
    * done preparing code history in 48s.
    * done fetching pull request and issue data in 8318s.
    * done pulling commit pull request info in 95s.
    * done collecting sample PR data in 214s.
    * exceeded max retry count 8 on https://api.github.com/repos/facebook/react/issues/comments?since=2013-05-29T20:20:53Z&sort=created&order=desc&per_page=100

Stat analysis took 270s for 111 pages.
Impact on the fetch when isolated is linear and Facebook and most repos are far from the 600 page limit.
Issue comments portion of the script seems likely to fail in general.
Variance in things that should have been affected is probably because of rate limiting.

**Pull request merged**